scripts/mk-compile-commands.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Can you move this file to the `mk/` directory? I think it's cleaner to keep all the build system related stuff in there.

Reply:
I did better and remove it altogether. I think it is small enought to fit in the Makefile now.